app/components/settings/project_custom_fields/header_component.html.erb

```diff
@@ -0,0 +1,36 @@
+
+<%=
+  component_wrapper do  
+    render Primer::OpenProject::PageHeader.new do |header|
+      header.with_title(variant: :default) { t('settings.project_attributes.heading') }
+      header.with_description { t('settings.project_attributes.heading_description') }
+      header.with_actions do
+        flex_layout(justify_content: :space_between, align_items: :center) do |action_buttons_container|
+          action_buttons_container.with_column(mr: 2) do
+            render(Primer::Beta::Button.new(
+              tag: :a,
+              href: new_admin_settings_project_custom_field_path(type: "ProjectCustomField"),
```

Comment:
I think we should disable this button if there are no `@project_custom_field_sections` are present and also validate the presence of the `ProjectCustomField#custom_field_section_id` field. Currently it is possible to create a project custom field without a section id, which is causing an error while rendering the overview page.
<details><summary>Demo of bug</summary>
<p>

![attribute missing](https://github.com/opf/openproject/assets/83396/0ba4c705-4bdf-40d8-8ecd-9c081a94156d)

</p>
</details>